webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
```

Comment:
Should this be "the RTCMediaStreamTrackStats whose trackIdentifier match the selector's id, and any RTCStats referencing that/those RTCMediaStreamTrackStats" or a list of specific stats (like you have now)?

The difference between that description and the list you described is you didn't include any RTCMediaStreamStats and there are stream stats with trackIds referencing RTCMediaStreamTrackStats. But these should be included?